### DISCUSSION TITLE
Changing resource names

### DISCUSSION THREAD

**Discussion by pcdiks** [Azure/ipam#59] [Q&A] [answered]
I changed the naming of azure resources to match the our naming convention.
I did this by updating the resourceNames section in the main.bicep.
This works great but I do see something strange with the cosmosdb container and database.
This is what I did with the cosmos items:
``` 
  cosmosContainerName: '${replace(rgName, '-', '')}ctr${padLeft(ipamIndex, 2, '0')}'
  cosmosDatabaseName: '${replace(rgName, '-', '')}${ShortName}db${padLeft(ipamIndex, 2, '0')}'
``` 

The container and database get created with the names I want but I still see an additional container and database
![image](https://user-images.githubusercontent.com/16719277/205266627-fd5f3a4a-496c-4d67-a8e4-9591a85ff526.png)

Should I add something to the App Service -> Configuration -> Application Settings?

**Comment by DCMattyG** [comment, marked as answer]
Hey @pcdiks, it's not something documented because I've only used this for dev testing at this point.

You can set alternate database/container names by adding these Application settings to your App Service:

DATABASE_NAME
CONTAINER_NAME

Hope that helps!

**Reply by pcdiks** [reply]
I added those settings to appService.bicep and that works. Thanks
